isotptun.c

```diff
@@ -118,14 +144,11 @@ int main(int argc, char **argv)
 	unsigned char buffer[BUF_LEN];
 	static char name[IFNAMSIZ] = DEFAULT_NAME;
 	int nbytes;
-
-	signal(SIGTERM, sigterm);
-	signal(SIGHUP, sigterm);
-	signal(SIGINT, sigterm);
+	int run_as_daemon = 1;
 
```

Comment:
Ok, then make it an `int`. Some tools use several levels of verbosity, this is why `verbose` might be an `int`.